core/src/main/java/hudson/model/Queue.java

```diff
@@ -1896,6 +1896,12 @@ default CauseOfBlockage getCauseOfBlockage() {
          */
         String getFullDisplayName();
 
+        /**
+         * Key for this task used for scheduling when searching for the next fee executor
```

Comment:
Typo: fee.

Also, rather difficult to understand. Perhaps rephrase for clarity?

Reply:
Thanks for pointing out the typo. I'll fix that.

How about something like the following for the javadoc:

        /**
         * Returns task-specific key which is used by the (@Link LoadBalancer) to choose one particular executor
         * amongst all the free executors on all possibly suitable nodes.
         * NOTE: To be able to re-use the same node during the next run this key should not change from one run to
         * another. You probably want to compute that key based on the job's name.
         * <p>
         * @return by default: (@Link #getFullDisplayName())
         *
         * @see hudson.model.LoadBalancer
         */

Reply:
LGTM.

Reply:
LGTM